packages/unchained-client/src/evm/bnbsmartchain/parser/__tests__/mockData/tokens.ts

```diff
@@ -1,5 +1,12 @@
 import { Token } from '../../../../../types'
 
+export const busdToken: Token = {
```

Comment:
This one actually makes sense since it is a `Token`, confirming the point that using `token` terminology for addresses might be confusing since this is also using the token terminology